CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Disdrodb contributing Guide
+
+Hi! Thanks for taking the time to contribute to disdrodb.
+
+You can contribute in many ways :
+
+ - Join the [discussion](https://github.com/ltelab/disdrodb/discussions)
+
```

Comment:
Do we want to create a slack channel?

Reply:
I would advocate to keep the discussions as close as possible to the code / in the same enviro. Gh discussions is really made for that